test/jdk/jdk/incubator/concurrent/StructuredTaskScope/StructuredTaskScopeTest.java

```diff
@@ -614,7 +614,7 @@ public void testJoinUntil4(ThreadFactory factory) throws Exception {
     public void testInterruptJoinUntil1(ThreadFactory factory) throws Exception {
         try (var scope = new StructuredTaskScope(null, factory)) {
             Future<String> future = scope.fork(() -> {
-                Thread.sleep(Duration.ofMillis(100));
+                Thread.sleep(Duration.ofMillis(1000));
```

Comment:
As a short term fix, this is okay, Duration.ofSeconds(2) might be a bit more robust. We are iterating on this API so there are test changes in the works, in this case we can change it to until that the sub-tasks is parked before calling joinUntil with the interrupt status set.

Reply:
So do I change to  `Duration.ofSeconds(2)` or cancel this PR?

Reply:
Go ahead with the change as this testInterruptJoinUntil isn't robust (as you've found with the -Xcomp runs). When we refresh the test we'll replace this part so that it poll the subtask until it is parked before testing joinUntil with the interrupt status set.